easybuild/tools/robot.py

```diff
@@ -394,6 +395,21 @@ def resolve_dependencies(easyconfigs, modtool, retain_all_deps=False, raise_erro
 
             additional = []
             for entry in easyconfigs:
+                _log.debug("Resolving dependencies for %s" % str(entry))
+                orig_build_opts = orig_robot_path = None
+                custom_build_opts = entry['ec'].custom_build_opts
+                if custom_build_opts is not None:
+                    _log.debug("Applying easyconfig specific options %s" % str(custom_build_opts))
+                    orig_build_opts = update_build_options(custom_build_opts)
+                    # Re-determine robot-path from options (borrowed from easybuild/tools/options)
```

Comment:
We should probably make a function out of this, rather than copying the code. That function could then be called whenever we need to calculate an updated `robot_path` (it could even update the `build_option` for us, but I'm not sure if we'd like that)